tools/codemod/codemod.ts

```diff
@@ -48,6 +60,8 @@ async function run(codemod: string, paths: string[], debug: boolean, optionOverr
   };
   debug && console.log("Running 'jscodeshift' with following configuration:", { transformFile, paths, options });
   await jscodeshift(transformFile, paths, options);
+
+  !skipCss && await runCssCodemod(args, transformFile);
```

Comment:
If both are promises and works on different files, could they be run at the same time ?

Reply:
We probably could, but let's keep them separated, easier to navigate output.